### ISSUE TITLE
Download Image

### ISSUE THREAD

**Issue by ramonsousa89** [OHIF/Viewers#674]
I can not find the option to enable Download as I had in version 1.
nor in article # 588

**Comment by swederik** [comment]
Yes, this has not been implemented in the React version. We would like to have the functionality again, but it is very low on the priority list. 

We would be happy to review a PR if you want implement it yourself. Here is the component in the Meteor version: https://github.com/OHIF/Viewers/tree/v1.x/Packages/ohif-viewerbase/client/components/viewer/downloadDialog

**Comment by dannyrb** [comment]
A "Download" tool now exists by default. You can find it in the toolbar for active cornerstone viewports.
